### DISCUSSION TITLE
jbspeakr.cc

### DISCUSSION THREAD

**Discussion by jbspeakr** [thuliteio/doks#1050] [Showcase]
Thanks for providing doks! 

I used it for my personal blog and tweaked its appearance slightly by introducing leads on blog posts, and setting up my own little bog post feature on the home page. everything works like a charm. big fan! 

https://www.jbspeakr.cc/

**Comment by h-enk** [comment]
Thanks for sharing. Nicely done!

**Comment by technosprout** [comment]
Newbie here, I like the blog site you created. Any chance you have the files on github?
